### PR TITLE
Fix terminology

### LINKED ISSUE #25: Helm charts and releases: inconsistent terminology?
Using Helm CLI shows that one Helm release can include several Helm charts - one main chart and several subcharts. Am I missing something or should `charts.yaml` therefore be `releases.yaml`?

Example: 

Generic chart: rails-app
Unique releases: dev-example-com, prod-example-com, dev-example-org, prod-example-org

Workflow:

1. Create a generic Helm chart with some default `values.yaml`
2. Add one or more releases, each with its own unique `values.yaml`

One cluster, many charts, more releases 🤔

“Helm deployment” here should be “Helm release” also. “Deployment” is a very specific Kubernetes term, and not a Helm term at all.

Am I wrong?

### PR DESCRIPTION
**Pull request by mumoshu**
In a backward compatible manner, so that (I believe) we could move the discussion made in #25 forward.

Possibly fixes #25

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -221,21 +222,28 @@ func before(c *cli.Context) (*state.HelmState, helmexec.Interface, error) {
 	quiet := c.GlobalBool("quiet")
 	kubeContext := c.GlobalString("kube-context")
 
-	state, err := state.ReadFromFile(file)
+	st, err := state.ReadFromFile(file)
+	if err != nil && strings.Contains(err.Error(), fmt.Sprintf("open %s:", DefaultHelmfile)) {
+		var fallbackErr error
+		st, fallbackErr = state.ReadFromFile(DeprecatedHelmfile)
```
Comment (sstarcher) [inline main.go]: when falling back to the deprecated value should we log a message to tell them it's deprecated?

Comment (mumoshu) [reply main.go]: Good point! I believe so.

Comment (mumoshu) [reply main.go]: Added to https://github.com/roboll/helmfile/pull/35/files#diff-7ddfb3e035b42cd70649cc33393fe32cR249

**Comment by mumoshu** [comment]
@sstarcher Thanks for reviewing 👍

**Pull request merged**